data/org.freedesktop.impl.portal.GlobalShortcuts.xml

```diff
@@ -130,11 +130,11 @@
         Emitted when a shortcut is activated.
     -->
     <signal name="Activated">
-      <arg type="o" name="session_handle" direction="in"/>
-      <arg type="s" name="shortcut_id" direction="in"/>
-      <arg type="t" name="timestamp" direction="in"/>
-      <arg type="a{sv}" name="options" direction="in"/>
-      <annotation name="org.qtproject.QtDBus.QtTypeName.In4" value="QVariantMap"/>
+      <arg type="o" name="session_handle" direction="out"/>
+      <arg type="s" name="shortcut_id" direction="out"/>
+      <arg type="t" name="timestamp" direction="out"/>
+      <arg type="a{sv}" name="options" direction="out"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.In3" value="QVariantMap"/>
```

Comment:
Is this right? You've given it 4 `out` arguments, and I infer from your other contributions of this annotation that they're using 0-based indexing, so I'd expect the annotation to be `.Out3` now?

(If I'm reading correctly, this was also an attempt at a silent bug-fix for Qt consumers because the annotation used to apply to a nonexistent 5th argument (`In4` in 0-based indexing), and was changed to point to the 4th argument?)

Reply:
@aleixpol, can you open a PR changing `In3` to `Out3`?

Reply:
https://github.com/flatpak/xdg-desktop-portal/pull/923

Good catch!

I was thinking it could make sense to run qtdbusxmltocpp in the CI to make sure it's all sound. Do you think it would be useful?

It would also need fixing some of the files that at the moment are not entirely working with it just yet.